examples/online/online-tracker.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const { createClient } = require('redis');
```

Comment:
You are not using this in this file. Take my other suggestion and move this logic into the `index.js` and you can just remove this line. Additionally, I dont think any of this code works because you are not calling your exported `createOnlineTracker` anywhere.